dynawo/sources/Models/Modelica/Dynawo/Examples/DynaFlow/IEEE14/Grid/IEEE14.mo

```diff
@@ -0,0 +1,230 @@
+within Dynawo.Examples.DynaFlow.IEEE14.Grid;
```

Comment:
Is it exactly the same as the DisconnectLine one but without the line disconnection ? if yes then the same remarks apply.
Also, this one could be also put in the TestCases folder and renamed IEEE14NoEvent for instance